predicates/forwarded/forwarded.go

```diff
@@ -0,0 +1,156 @@
+/*
+Package forwarded implements a set of custom predicate to match routes
+based on the standardized Forwarded header.
+
+https://datatracker.ietf.org/doc/html/rfc7239
+https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Forwarded
+
+Examples:
+
+    // only match requests to "example.com"
+    example1: ForwardedHost("example.com") -> "http://example.org";
+
+    // only match requests to http
+    example2: ForwardedProtocol("http") -> "http://example.org";
+
+    // only match requests to https
+    example3: ForwardedProtocol("https") -> "http://example.org";
+*/
+package forwarded
+
+import (
+	"github.com/zalando/skipper/predicates"
+	"github.com/zalando/skipper/routing"
+	"net/http"
+	"regexp"
+	"strings"
+)
+
+const (
+	NameHost  = "ForwardedHost"
+	NameProto = "ForwardedProtocol"
+)
+
+const (
+	HTTP  string = "http"
+	HTTPS string = "https"
+)
```

Comment:
Is there a need for this public constants, maybe just inline them?

Reply:
I'll inline them.